fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java

```diff
@@ -385,7 +386,7 @@ private static void checkReplicationNum(short replicationNum) throws AnalysisExc
         // compute-storage-separation table will be created and in this case the replication_num will
         // be ignored, so there is no need to check whether the number of alive nodes is greater than the
         // replication_num.
-        if (GlobalStateMgr.getCurrentState().isSharedNothingMode()) {
+        if (!RunMode.getCurrentRunMode().isIgnoreReplicationNum()) {
```

Comment:
create lake table in HYBRID run mode does not need to check replication number.

Reply:
Yes I know, need some refactoring here. Will do that in a separate patch.